projects/js-packages/publicize-components/src/components/share-buttons/useAvailableNetworks.tsx

```diff
@@ -0,0 +1,32 @@
+import { SocialIconSlug } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+
+type Network = {
+	label: string;
+	networkName: SocialIconSlug;
+	url: string;
+};
+
+/**
+ * Returns the list of available networks.
+ *
+ * @returns {Array<Network>} The list of available networks.
+ */
+export function useAvailableNetworks() {
```

Comment:
What's the advantage to having this as a hook with a memoized return? Could this just be a static array?

Reply:
It's for i18n which will break on static arrays because the language is initiated way after the modules are loaded.

Reply:
I see. Could we pass it through `__` as it's rendered instead?

Reply:
Then i18n string extractor build plugins will not work.